README.md

```diff
@@ -127,24 +127,37 @@ Consider using [rlink] instead of `yarn link`
 
 ### Tests
 
-Tests are run by [mocha] under the hood, and written using [chai] and [sinon]. You can easily run the tests suite with:
+Tests are run by jest under the hood. You can easily run the tests suite with:
 
 ```sh
 $ cd cozy-drive
 $ yarn test
 ```
 
-:pushpin: Don't forget to update / create new tests when you contribute to code to keep the app the consistent.
+:pushpin: Don't forget to update / create new tests when you contribute to prevent any regression.
 
+### Release & version bumping
 
-### Open a Pull-Request
+To release a new version, [follow the release guide](https://docs.cozy.io/en/cozy-app-publish/#release-workflow).
```

Comment:
This document uses link anchor style links elsewhere so you should avoir mixed-style links in the same document IMHO